go/vt/tabletmanager/healthcheck.go

```diff
@@ -259,6 +259,9 @@ func (agent *ActionAgent) runHealthCheck(targetTabletType pbt.TabletType) {
 	stats := &pb.RealtimeStats{
 		SecondsBehindMaster: uint32(replicationDelay.Seconds()),
 	}
+	blpStats := agent.BinlogPlayerMap.StatusSummary()
+	stats.SecondsBehindMasterFilteredReplication = blpStats.maxSecondsBehindMaster
```

Comment:
For discussion: Instead of adding filtered replication fields to RealTimeStats, we could add a new message for filtered replication only at the same level as RealTimeStats.

BinlogPlayerMap.StatusSummary() could return this proto3 directly.

What do you think?